astropy/coordinates/builtin_frames/galactocentric.py

```diff
@@ -217,50 +217,52 @@ def validate(cls, value):
 
     To transform to the Galactocentric frame with the default
     frame attributes, pass the uninstantiated class name to the
-    ``transform_to()`` method of a coordinate frame or
-    `~astropy.coordinates.SkyCoord` object::
+    ``transform_to()`` method of a `~astropy.coordinates.SkyCoord` object::
 
         >>> import astropy.units as u
         >>> import astropy.coordinates as coord
-        >>> c = coord.ICRS(ra=[158.3122, 24.5] * u.degree,
-        ...                dec=[-17.3, 81.52] * u.degree,
-        ...                distance=[11.5, 24.12] * u.kpc)
+        >>> c = coord.SkyCoord(ra=[158.3122, 24.5] * u.degree,
+        ...                    dec=[-17.3, 81.52] * u.degree,
+        ...                    distance=[11.5, 24.12] * u.kpc,
+        ...                    frame=coord.ICRS)
```

Comment:
I think we can leave off the `frame=coord.ICRS` and (optionally) add a comment like `# the default frame is ICRS`.

Reply:
Explicit is better than implicit? :wink:

Reply:
OK, but then we should [add it everywhere](https://docs.astropy.org/en/latest/coordinates/index.html) and remove the default!?

Reply:
I have always thought it was weird there was a default, but that ship has sailed off to another solar system at this point.

Reply:
I'd leave it out (like we don't use `scale='utc'` in `Time`) or use the string 'icrs'.

Reply:
Changed it to `frame='icrs'`